README.md

```diff
@@ -180,6 +180,22 @@ var client = new presto.Client({
 });
 ```
 
+## Development
+
+When working on this library, you can use the included docker-compose.yml file to spin up a Presto and Trino DBs, which can be done with:
+
+```
+docker compose up
+```
```

Comment:
As I commented (https://github.com/tagomoris/presto-client-node/pull/69#issuecomment-1573343931), I want a note that we should wait for the servers' launch completion:

```
presto-client-node-trino-1   | 2023-06-02T08:12:37.760Z	INFO	main	io.trino.server.Server	======== SERVER STARTED ========
presto-client-node-presto-1  | 2023-06-02T08:13:29.760Z	INFO	main	com.facebook.presto.server.PrestoServer	======== SERVER STARTED ========
```

Reply:
So sorry, added in 3365b571f1c15b51d8c0e62ad7c6d6a3c172d7b2.